bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h

```diff
@@ -1325,7 +1400,7 @@ namespace ElectionGuardInterop
             return this->_instance->isValid();
         }
 
-          property array<Byte> ^
+        property array<Byte> ^
```

Comment:
White space change?

Reply:
That line was not changed, so not sure why the whitespace changed.